Makefile

```diff
@@ -44,6 +44,26 @@ ifeq ($(TEST_TYPE),coverage)
 		fi; \
 	done
 endif
+ifeq ($(TEST_TYPE),fuzzit)
+	# skip fuzzing for PR
+	if [ "$(TRAVIS_PULL_REQUEST)" = "false" ] || [ "$(FUZZIT_TYPE)" = "local-regression" ] ; then \
+        export GO111MODULE=off; \
+        go get -u github.com/dvyukov/go-fuzz/go-fuzz-build; \
+        go get -u -v .; \
+        cd ../../go-acme/lego; \
+        git checkout v2.5.0; \
```

Comment:
oh wow, this is pretty horrible :/

I think it make sense to make this into Makefile.fuzz under some phony install target, so we can call `make -f Makefile.fuzz install` here. or $(MAKE) or somethign

Reply:
This is the workaround for the strange glitch with go-acme v3. How do you suggest working around that? (not sure I got it regarding the make install)